runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -178,6 +178,31 @@ task validatesRunnerLegacyWorkerTest(type: Test) {
   }
 }
 
+task validatesRunnerLegacyWorkerJava11Test(type: Test) {
+    group = "Verification"
+    dependsOn ":beam-runners-google-cloud-dataflow-java-legacy-worker:shadowJar"
+    systemProperty "beamTestPipelineOptions", JsonOutput.toJson([
+            "--runner=TestDataflowRunner",
+            "--project=${dataflowProject}",
+            "--tempRoot=${dataflowValidatesTempRoot}",
+            "--dataflowWorkerJar=${dataflowLegacyWorkerJar}",
+    ])
+    
+    systemProperty "java.specification.version", "11"
```

Comment:
Please confirm if I understand correctly: this task runs the "validatesRunner" suite with Java11. Previously everything gets built with java 8 because this line sets the Java11 version AFTER task `:beam-runners-google-cloud-dataflow-java-legacy-worker:shadowJar` has been invoked. Am I right here?

Reply:
This java specification system property has no effect on compilation, it's used only in `DataflowRunner.java` to select proper worker harness image to pull from GCR. This task runs the suite on Dataflow using a Java 11 worker harness - this setting controls only the harness image

Reply:
👍 thanks for clarifying